pkg/auth/providers/ldap/ldap_client.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 var operationalAttrList = []string{"1.1", "+", "*"}
+var nestedGroupPrincipals []v3.Principal
```

Comment:
this cannot be a global variable. It is used on each login request. This isnt safe for use by multiple requests at once.